codespell_lib/data/dictionary.txt

```diff
@@ -10361,6 +10381,9 @@ detinations->destinations
 detremental->detrimental
 detremining->determining
 detrmine->determine
+detrmined->determined
+detrmines->determinse
```

Comment:
Suggested word `detrmines->determinse` is a misspell word.
```text
detrmines->determines
```

Reply:
[323c5f5](https://github.com/codespell-project/codespell/pull/2418/commits/323c5f5af46edb2f10393ca2f0ebc85f29989c6a)